### PR TITLE
Feature/remarkable plugins

### PR DESCRIPTION
**Pull request by RubenSmit**
I saw the discussion in this issue https://github.com/johnotander/ember-remarkable/issues/7 and decided to give it a go and fix it myself. Never done this kind of thing before, so I would like to see your reactions. This fixes all the problems in the issue mentioned before.

### CODE REVIEW

File: addon/components/md-text.js [hunk 1]
```diff
@@ -58,6 +59,16 @@ export default Ember.Component.extend({
       ]);
     }
 
+    let config = Ember.getOwner(this).resolveRegistration('config:environment');
+    if (config.emberRemarkable.plugins !== false) {
```
Comment (johnotander) [inline addon/components/md-text.js]:
```js
if (config.emberRemarkable.plugins)
```

File: addon/components/md-text.js [hunk 2]
```diff
@@ -58,6 +59,16 @@ export default Ember.Component.extend({
...
+      let usePlugins = this.get('usePlugins');
+      config.emberRemarkable.plugins.forEach(function(plugin) {
+        if (usePlugins == null || usePlugins.indexOf(plugin.name) !== -1) {
```
Comment (johnotander) [inline addon/components/md-text.js]:
```js
if (!Ember.isEmpty(usePlugins) && usePlugins.indexOf(plugin.name) !== -1)
```

**Comment by johnotander** [comment]
Rad, looking good! Thanks for contributing! I've added a couple comments to your PR so far.

It looks like for some reason node v0.12 (in Travis CS) isn't happy with the way the configuration is being grabbed. Though, I'm not sure why? I will have some time this weekend to look into it if you don't get a chance before then.

Before I can merge we will need to add a test and docs, but I absolutely love where this is headed. 👯 🍻 😍

**Comment by RubenSmit** [comment]
Thanks for the comments! I will look at them later this week.

**Comment by johnotander** [comment]
Thank you so much for your work on this, we've gone ahead and added this functionality with a different PR so I will close this now.

**Pull request closed without merging**